lib/msf/core/exploit/remote/smb/relay/ntlm/target/http/client.rb

```diff
@@ -84,7 +84,7 @@ def relay_ntlmssp_type3(client_type3_msg)
       )
       res = @client.send_recv(req, @timeout, true)
 
-      if res.code.between?(200, 299)
+      if res.code.between?(200, 299) || res.code == 403
```

Comment:
The response code which indicates the NTLM relay was successful needs to be configurable from the module. Stay tuned.